WooCommerce/UITestsFoundation/Screens/Orders/OrdersScreen.swift

```diff
@@ -37,4 +37,18 @@ public final class OrdersScreen: ScreenObject {
         searchButton.tap()
         return try OrderSearchScreen()
     }
+
+    public func verifyOrdersList(orders: [OrderData]) throws -> Self {
+        XCTAssertEqual(orders.count, app.tables.cells.count, "Expecting \(orders.count) orders, got \(app.tables.cells.count) instead!")
+        app.assertTextVisibilityCount(textToFind: String(orders[0].id))
+        app.assertTextVisibilityCount(textToFind: String(orders[0].total))
+        app.assertTwoTextsAppearOnSameLabel(firstSubstring: String(orders[0].id), secondSubstring: orders[0].billing.first_name)
```

Comment:
This was [renamed/simplified recently](https://github.com/woocommerce/woocommerce-ios/blob/a1da511efd34f363cf2793d2766af77e8897df47/WooCommerce/UITestsFoundation/XCTest%2BExtensions.swift#L147) to `assertLabelContains`, I imagine this would fail after merging to trunk since it doesn't exist on that branch anymore.

Reply:
Yep, it did! Sorted, thanks :)